.github/workflows/quality.yml

```diff
@@ -117,6 +117,28 @@ jobs:
         if: steps.files.outputs.all_changed_files_count > 0
         uses: ./.github/composite-actions/a11y
 
+  a11y-with-playwright:
```

Comment:
Does this mean this will run on every PR? If that’s the case, I’m not sure it’s called for, since they’re accessibility tests rather than functional tests.

Maybe we could run them like cron jobs, twice a week or something like that, wdyt?

Reply:
Nope, that is the whole point of this: https://github.com/adevinta/spark/pull/2324/files#diff-b31f1e5654d027d84c24cdba2ff31ab79e3ea617091ed348a97476a27b076078R139 :+1: 
Playwright tests (just as initial a11y one) would only be triggered on specific file changes. Else it would skip.